docker-entrypoint.sh

```diff
@@ -40,6 +40,10 @@ download_and_configure_ilitool ilivalidator $ILIVALIDATOR_VERSION $ILITOOLS_HOME
   cp -f $ILICHECK_WEB_ASSETS_DIR/* $ILICHECK_APP_HOME_DIR/ClientApp/build/ && \
   echo "done!"
 
+# Use default user:group if no $PUID and/or $PGID is provided.
+groupmod -o -g ${PUID:-1654} app && \
+  usermod -o -u ${PGID:-1654} app &> /dev/null
```

Comment:
q: Den Default `1654` verstehe ich nicht. Weshalb soll das der Default sein?
Kann die Modifikation die Annahmen von  .Net oder Docker brechen? Gibt es im .net 8 image keine `USER app` im Dockerfile? -> Würde bedeutet, dass wir im Entrypoint auch keine Root Berechtigungen haben. https://www.docker.com/blog/understanding-the-docker-user-instruction/

Reply:
Meine Idee damit war, dass so ohne eine Anpassung an den env-Variabeln die id des `app` Users aus dem .NET Image nicht geändert wird.
`USER app` ist nur eine Empfehlung (keine Vorgabe) von .NET 8 und könnte ev. auch durch das spätere Anpassen der id brechen, aber wir müssen den Entrypoint als `root` ausführen für `chown` und das Logging des cron Jobs.